.github/workflows/ci.yaml

```diff
@@ -18,6 +18,7 @@ env:
   ARBITRUM_ONE_RPC_URL: ${{ secrets.ARBITRUM_ONE_RPC_URL }}
   SEPOLIA_RPC_URL: ${{ secrets.SEPOLIA_RPC_URL }}
```

Comment:
(arbitrary comment)

Should this pr also add base to the wallet dropdown.

Reply:
Think we have that covered in this [PR](https://github.com/ScopeLift/umbra-protocol/pull/643)